polylogue/core/config_validation.py

```
@@ -23,6 +23,46 @@ def expand(self) -> Dict[str, Path]:
+
+    model_config = ConfigDict(extra="forbid")
+
+
+class IndexModel(BaseModel):
+    backend: Optional[str] = None
+    qdrant: Optional[QdrantModel] = None
+
+    model_config = ConfigDict(extra="forbid")
+
+    @model_validator(mode="after")
+    def ensure_backend(cls, values: "IndexModel") -> "IndexModel":
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)